tests/test_activate.py

```diff
@@ -2940,112 +2942,127 @@ def test_powershell_PATH_management(shell_wrapper_integration: tuple[str, str, s
         prefix = join(prefix, "envs", "test")
         print("## [PowerShell activation PATH management] Starting test.")
         shell.sendline("(Get-Command conda).CommandType")
-        shell.p.expect_exact("Alias")
+        shell.expect_exact("Alias")
         shell.sendline("(Get-Command conda).Definition")
-        shell.p.expect_exact("Invoke-Conda")
+        shell.expect_exact("Invoke-Conda")
         shell.sendline("(Get-Command Invoke-Conda).Definition")
-        shell.p.expect(".*\n")
+        shell.expect(".*\n")
 
         shell.sendline("conda deactivate")
         shell.sendline("conda deactivate")
 
         PATH0 = shell.get_env_var("PATH", "")
         print(f"PATH is {PATH0.split(os.pathsep)}")
         shell.sendline("(Get-Command conda).CommandType")
-        shell.p.expect_exact("Alias")
+        shell.expect_exact("Alias")
         shell.sendline(f'conda create -yqp "{prefix}" bzip2')
         shell.expect("Executing transaction: ...working... done.*\n")
 
 
+def _contains(prefix: str, path: str) -> bool:
+    try:
+        Path(path).relative_to(prefix)
+        return True
+    except ValueError:
+        # ValueError: path is not relative to prefix
+        return False
+
+
+STRIPPED_PATH = os.pathsep.join(
+    path
+    for path in os.environ["PATH"].split(os.pathsep)
+    if not _contains(context.conda_prefix, path)
+)
+
+
 @pytest.mark.skipif(not which("cmd.exe"), reason="cmd.exe not installed")
 @pytest.mark.integration
-def test_cmd_exe_basic_integration(shell_wrapper_integration: tuple[str, str, str]):
+def test_cmd_exe_basic_integration(
+    shell_wrapper_integration: tuple[str, str, str],
+    monkeypatch: MonkeyPatch,
+):
     prefix, charizard, _ = shell_wrapper_integration
+    conda_bat = Path(CONDA_PACKAGE_ROOT, "shell", "condabin", "conda.bat")
+    monkeypatch.setenv("PATH", STRIPPED_PATH)
```

Comment:
`monkeypatch.setenv` instead of `env_vars`

Reply:
Don't we use conda env_vars because it resets the context? Will that be a problem compared to monkeypatch?

Reply:
I've been trying to move away from `env_var(s)?` in general

In this case resetting the context has no impact since the context doesn't do anything with `PATH`